src/configuration/app/firewall.md

```diff
@@ -0,0 +1,63 @@
+# Outbound firewall
+
+In some situations, compliance regulations may require you to limit outbound traffic from your application.  The `firewall` property allows you to do so.
+
+This setting has no impact on inbound requests to your application.  For that, use the environment access control settings in the Management Console.
+
+## Syntax
+
+The `firewall` property defines one or more whitelist entries for outbound requests.  It's basic syntax is as follows:
+
+```yaml
+firewall:
+  outbound:
+    - protocol: tcp
+      ips: ["1.1.1.1/32"]
+      ports: [443]
+    - protocol: tcp
+      ips: ["1.2.3.4/32"]
+      ports: [443]
+```
+
+The above example allows two outbound rules over TCP.  All other outbound requests will be blocked and will time out eventually (usually after 30 seconds).
+
+If no rules are specified, the default `firewall` configuration is equivalent to:
+
+```yaml
+firewall:
+  outbound:
+    - protocol: tcp
+      ips: ["0.0.0.0/0"]
+```
+
+That is, all outbound TCP traffic is allowed on all ports (aside from port 25, which is always blocked without exception).  In the majority of cases the default is sufficient for most applications.
+
+## Options
+
+Each firewall rule has three configuration values.
+
+### `protocol`
+
+The protocol value is always `tcp`.  Outbound UDP request are not allowed anyway.  As a result this property can be omitted in virtually every circumstance.
```

Comment:
the default value is "tcp" also, so you don't have to specify it. The "are not allowed anyway" part sounds a bit condescending :)